### ISSUE TITLE
Bag Items

### ISSUE THREAD

**Issue by cawk** [AnthonyOSX/GO-Bot#130]
Hello can you add an option not to discard potions and discard pokeballs and keep higher balls only or give the option not to discard at all 

i would add this to the source code myself but ive recently restored my computer and waiting on the windows 10 upgrade to download before i install visual studio again 

if you havent added a feature like this before my computer is set up properly ill add it myself and make a pull request if that is okay?

**Comment by jodriscoll** [comment]
I'm confused...

Can you just fork the repo, make your code changes, submit a pull request and @AnthonyOSX can review and approve/reject it?

...?

**Comment by cawk** [comment]
yes i could but i have recently reinstalled my computer to windows 7 (only had that disc) but now using the windows 10 upgrade but it has to download first so waiting for that to happen first then ill install visual studio 

but i know what you mean make the changes on here but first i would have to debug my changes to make sure they work before i push them out 

but like i said i will do it later if its not done my upgrade is nearly complete but we all know visual studio takes its time to install so will do it then

**Comment by filipst6** [comment]
He said that update will be pushed later with more refined discard system in place.

**Comment by fenixasin** [comment]
person is a hell of a coder to not need much help from others...

**Comment by cawk** [comment]
and i dont doubt that but if he didnt have time for example i could of done it for him 

i only offered as it was an issue myself and some others had the same issue
im sure the creator wouldnt mind if i did add some code that i shared with others even if he didnt want to use it

**Comment by filipst6** [comment]
Well you can do it. I'm pretty sure that source code wouldn't be available if developer minded you forking it.

**Comment by estrapou** [comment]
You're inconsistent when you write cawk, it's not clear. However, I've requested a feature that would allow customization of the discarded items as well in a previous post.  So basically to be able to choose how many items we want to keep or if we want to auto delete items knowing that the backpack can hold minimum 350 items and up to 1000 items, hopefully the feature will be available soon.
